src/test/java/duckClassTest.java

```diff
@@ -0,0 +1,136 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+import java.util.ArrayList;
+import java.util.Collections;
+import static org.testng.Assert.assertTrue;
+
+public class duckClassTest extends settingClass {
+
+    @Test
+    public void duckClickHeaderRubberDarkTest() {    //WPath locators
+        String title;
+        driver.get("https://litecart.stqa.ru/en/");
+        driver.findElement(By.xpath("//nav[@id='site-menu']/ul/li[@class='category-1']")).click();
+        title=driver.findElement(By.xpath("//div[@id='box-category']/h1[@class='title']")).getText();
+        assertTrue(title.contains("Rubber Ducks"),"Rubber Ducks not found");
+    }
+
+    @Test
+    public void duckClickHeaderDeliveryInformationTest() {  // CSS locators
+        ArrayList<WebElement> listElements;
+        String footer = "";
+
+        ArrayList<String> listTitle = new ArrayList<>();
+        driver.get("https://litecart.stqa.ru/en/");
+        listElements = new ArrayList<>(driver.findElements(By.cssSelector("#site-menu > ul > li")));
+        for( WebElement element : listElements){
+            if (element.getText().equals("Delivery Information")) {
+                element.click();
+                break;
+            }
+        }
+        listElements = new ArrayList<>(driver.findElements(By.cssSelector("#breadcrumbs > ul > li")));
+        for( WebElement element : listElements){
+            footer = footer + " " + (element.getText());
+        }
+        assertTrue(footer.contains("Delivery Information"),"Delivery Information not found");
+    }
+
+    @Test
+    public void duckClickHeaderTermConditionTest() {    //WPath locators
+        ArrayList<WebElement> listElements;
+        String footer = "";
+
+        driver.get("https://litecart.stqa.ru/en/");
+        driver.findElement(By.xpath("//li[@class='page-4']")).click();
+        listElements = new ArrayList<>(driver.findElements(By.xpath("//nav[@id='breadcrumbs']/ul/li")));
+        for (WebElement element :  listElements){
+            footer = footer + " " + (element.getText());
+        }
+        assertTrue(footer.contains("Terms & Conditions"),"Terms & Conditions not found");
+    }
+
+    @Test
+    public void duckSubcategoryTest() {     // Actions
```

Comment:
What exactly are we testing here? I'm not sure this is a good scenario + the name doesn't really say a lot about the subject of testing